smoketest.bash

```diff
@@ -189,7 +183,7 @@ done
 
 if [ "${DRY_RUN}" = "true" ]; then
     set +xe
-    docker-compose \
+    "${container_engine}" compose \
```

Comment:
is there any reason to have both `ce` and `container_engine` variables? Could we replace `ce` with just `container_engine` from the start?